pkg/controller/jenkins/configuration/user/seedjobs/seedjobs.go

```diff
@@ -164,6 +164,14 @@ func (s *SeedJobs) EnsureSeedJobs(jenkins *v1alpha2.Jenkins) (done bool, err err
 		if err != nil {
 			return false, err
 		}
+
+		requeue, err := s.waitForSeedJobAgent(AgentName)
+		if err != nil {
+			return false, err
+		}
+		if requeue {
+			return false, nil
+		}
```

Comment:
also, there's an old regression in line 193 / 201 starting with "if done && ..."